src/App.js

```diff
@@ -46,11 +46,10 @@ class App extends Component {
       <div className="App">
         <div className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
```

Comment:
The logo var on this line is imported at the top of the file. Try importing the q2 version.